src/fw-update/fw-update-factory.cpp

```diff
@@ -50,10 +54,17 @@ namespace librealsense
                 auto usb = platform::usb_enumerator::create_usb_device(info);
                 if (!usb)
                     continue;
-                if (ds::RS_RECOVERY_PID == info.pid)
-                    return std::make_shared< ds_update_device >( shared_from_this(), usb );
-                if (ds::RS_USB2_RECOVERY_PID == info.pid)
-                    return std::make_shared< ds_update_device >( shared_from_this(), usb );
+                switch( info.pid )
+                {
+                case ds::RS_D400_RECOVERY_PID:
+                case ds::RS_D400_USB2_RECOVERY_PID:
+                    return std::make_shared< ds_d400_update_device >( shared_from_this(), usb );
+                case ds::RS_D500_RECOVERY_PID:
```

Comment:
I think this shouldn't be in dev...

Reply:
Don't you think D555e USB will be the same?
I think it will be..
But I will remove and you can decide in the future

Reply:
Wait, you want me to remove the definition and all references as well?
Or just this line?

Reply:
I don't this PID should be on dev.

Reply:
Removed